pkgs/development/compilers/halide/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, stdenv, fetchFromGitHub, llvm }:
+
+stdenv.mkDerivation {
+  name = "halide-2015.10.22";
+
+  src = fetchFromGitHub {
+    owner = "halide";
+    repo = "Halide";
+    rev = "release_2015_10_22";
```

Comment:
You could bind the version to a name and use it here by replacing . with _. This is not mandatory but would reduce the number of lines that need to change on an update.